vue.config.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
   lintOnSave: false,
+
+  transpileDependencies: [
+    'vuetify',
```

Comment:
I have no idea if this is necessary in our current tooling. It was added by `vue add vuetify`.

Reply:
https://cli.vuejs.org/config/#transpiledependencies

Since we don't have babel, it just doesn't do anything.  I think it's probably good to leave in incase someone wants to add babel.  Maybe add a comment about it.